example/fittemplate.md

```diff
@@ -0,0 +1,157 @@
+
+## Introduction
+
+A volume conduction model of the head is required for source reconstruction. Ideally you base the
+headmodel on the individual's anatomical MRI, but that is not always available. In the case of EEG you can use a template head model and fit your measured electrodes (or template electrodes) on the scalp of the template model. For MEG you cannot simply use a template head model, since the distance between the head and the (fixed) MEG sensors depends on the head size.
+
+In this example we will show two ways on how to create an individual head model on the basis of surface
+data acquired with the Polhemus. Both methods determine the translation, rotation and scaling of the
+template head model to fit the Polhemus head shape, but the second method applies a more detailed spatial scaling.
+
+## Download
+
+Both approaches will be demonstrated using the same dataset, which you can find on the
+[ftp server](ftp://ftp.fieldtriptoolbox.org/pub/fieldtrip/tutorial/epilepsy). More
+information on this dataset can be found [here](/tutorial/epilepsy/).
+
+## Loading the data
+
+Before starting with FieldTrip, it is important that you set up your
+[MATLAB path](/faq/should_i_add_fieldtrip_with_all_subdirectories_to_my_matlab_path) properly.
+
+You can load the head shape measured during the MEG recording with the Polhemus and a
+template volume conduction model. We have to ensure that they have consistent units, hence
+we will convert the units into mm.
+
+    polhemus = ft_read_headshape('epilepsy/case1/ctf_data/case1.pos');
+    polhemus = ft_convert_units(polhemus,'mm');
+
+    template = ft_read_vol('standard_bem.mat');
+    template = ft_convert_units(template,'mm');
+
+## Coregistration
+
+In the next step we coregister the template head model with the Polhemus head shape to ensure that they are expressed consistently in the MEG coordinate system (i.e. relative to the same origin and with the axes pointing in the same direction). The coregistration consists of a rotation and translation.
+
+    cfg = [];
+    cfg.template.headshape      = polhemus;
+    cfg.checksize               = inf;
+    cfg.individual.headmodel    = template;
+    cfg                         = ft_interactiverealign(cfg);
+    template                    = ft_transform_geometry(cfg.m, template);
+
+## Removing unshared details between Polhemus and the template
+
+To create the individualized head model it is important that the scalp
+surface of the template head model only contains features that are also in the Polhemus head
+shape, and vice versa. We can use **[ft_defacemesh](/reference/ft_defacemesh)** to remove some features.
+
+So first we visualise both meshes:
+
+    ft_plot_mesh(template.bnd);  
+    ft_plot_mesh(polhemus);  
+
+{% include image src="/assets/img/example/fittemplate/template_surface_polhemus_surface_plot.png" %}
+
+As the template model and the Polhemus measurement have exclusive features, we will now remove these  
```

Comment:
Please describe what you mean with "exclusive features". Looking at the figures I would say that you have removed a chunk at the back of the template that was not scanned with the polhemus

Reply:
Clarified this.